v1/ast/policy.go

```diff
@@ -914,14 +869,11 @@ func NewHead(name Var, args ...*Term) *Head {
 	return head
 }
 
-// VarHead creates a head object, initializes its Name, Location, and Options,
-// and returns the new head.
-func VarHead(name Var, location *Location, jsonOpts *astJSON.Options) *Head {
+// VarHead creates a head object, initializes its Name and Location and returns the new head.
+// NOTE: The JSON options argument is no longer used, and kept only for backwards compatibility.
+func VarHead(name Var, location *Location, _ *astJSON.Options) *Head {
```

Comment:
This might go against what might be otherwise be considered idiomatic go, but we could also make the `*astJSON.Options` arg variadic, so new code isn't required to use it, and maybe make it easier to drop in the future.